MaintenancePackages.sln

```diff
@@ -172,6 +148,12 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TDS.Servers", "src\System.D
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Address", "src\System.Data.SqlClient\tests\ManualTests\SQL\UdtTest\UDTs\Address\Address.csproj", "{D1392B54-998A-4F27-BC17-4CE149117BCC}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.IO.Redist", "Microsoft.IO.Redist", "{A376FF60-3E89-40F1-BD79-652C4DD6EE89}"
```

Comment:
I didn't move specifically, I removed empty test related folders so that might have regenerated this?? I can see if I can revert these specific changes to Microsoft.IO.Redist